### ISSUE TITLE
Setting up the git filter using `.gitattributes` gives `Installation failed: git is not on path!` error

### ISSUE THREAD

**Issue by devmcp** [kynan/nbstripout#155]
Attempting to install with:

```
nbstripout --install --attributes .gitattributes
```

gives the following error:

```
Installation failed: git is not on path!
```

even though git is installed and is functioning fine.

**Comment by kynan** [comment]
Which version of `nbstripout` are you using? Which platform?

**Comment by gunar** [comment]
Same here.

Linux zeno 5.4.123-1-MANJARO 
git version 2.31.1
nbstripout==0.4.0

**Comment by gunar** [comment]
Btw, I'm using `pipenv`. Not working with `pipenv run nbstripout --install …` nor `pipenv shell`

**Comment by devmcp** [comment]
I'm using a `conda` environment on macOS 11.4
git version 2.32.0
nbstripout==0.4.0

**Comment by kynan** [comment]
Thanks for reporting, I can reproduce the issue. I'll release a fix in 0.4.1. Meanwhile, as a workaround please use

    nbstripout --install --attributes $PWD/.gitattributes
